### PR TITLE
Bridge V2 view minor UI improvements

### PR DESCRIPTION
**Pull request by nikarm22**
Tickets:
[ETA formatting](https://www.notion.so/wormholelabs/UI-FEATURE-Prettify-ETA-formatting-b3cab769f91747d2ae44271e8d166826)
[Token decimals](https://www.notion.so/wormholelabs/UI-FEATURE-Can-we-truncate-all-token-amounts-to-4-decimals-8eff447f7c704c9fbc753f7a99ca6e42)

UI:
Screenshot from 2024-09-05 18-02-31
Screenshot from 2024-09-05 18-03-12

### CODE REVIEW

File: wormhole-connect/src/views/v2/Redeem/index.tsx [hunk 1]
```diff
@@ -23,13 +23,10 @@ import PoweredByIcon from 'icons/PoweredBy';
...
 import { setRoute } from 'store/router';
-import { displayAddress } from 'utils';
+import { displayAddress, millisToHumanString } from 'utils';
 import { interpretTransferError } from 'utils/errors';
 import { joinClass } from 'utils/style';
-import {
-  millisToMinutesAndSeconds,
-  minutesAndSecondsWithPadding,
-} from 'utils/transferValidation';
+import { minutesAndSecondsWithPadding } from 'utils/transferValidation';
```
Comment (emreboga) [inline wormhole-connect/src/views/v2/Redeem/index.tsx:29]: Unused?

**Review by emreboga — APPROVED** [review]
LGTM with one minor comment.

File: wormhole-connect/src/views/v2/Redeem/index.tsx (hunk 1 above)
Comment (nikarm22) [reply wormhole-connect/src/views/v2/Redeem/index.tsx:29]: Used for countdown.

**Comment by Wesleyleung** [comment]
@nikarm22 I still notice this issue on the transaction status view. 
Screenshot 2024-09-06 at 5 31 38 PM

**Pull request merged**